model-optimizer/extensions/middle/ONNXResize11ToInterpolate.py

```diff
@@ -34,12 +34,13 @@ def replace_resize(graph: Graph, resize: Node):
         log.warning('The input shape is not 4D or 5D for op with name {}'.format(resize_name))
         return
 
-    num_of_inputs = len([port for port in resize.in_ports().values() if not port.disconnected()])
-    assert num_of_inputs in {3, 4}, \
-        "Number of inputs of ONNXResize (with name {}) should be equal to 3 or 4".format(resize_name)
+    assert (resize.is_in_port_connected(0) and (resize.is_in_port_connected(2) or resize.is_in_port_connected(3))), \
+        "Scales or sizes inputs must be connected to Node {} with op {}.".format(resize.soft_get("name", resize.id),
+                                                                                 resize.soft_get("op", resize.id))
```

Comment:
Why you use `resize.soft_get("op", resize.id)` here and below?
Theoretically, if a node does not have an `op` attribute, then the function will return `id` of this node. I suppose it is incorrect. This is a minor note, because node always have `op` attribute, but I think we should avoid such cases to make code clear

Reply:
I am not sure that we need change that.

Reply:
The comment from @yekruglov  was correct. `resize.soft_get("op", resize.id)` looks awkward. Please, fix.